backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceCandBL.java

```diff
@@ -2462,4 +2463,20 @@ public void setAsyncBatch(@NonNull final InvoiceCandidateId invoiceCandidateId,
 
 		invoiceCandDAO.save(invoiceCandidate);
 	}
+
+	@Override
+	public Quantity getOrderedQtyStockUOM(@NonNull final I_C_Invoice_Candidate ic)
+	{
+		final I_C_UOM productUOM = productBL.getStockUOM(ic.getM_Product_ID());
+
+		return Quantity.of(ic.getQtyOrdered(), productUOM);
```

Comment:
this is OK, but FYI: you could also use `de.metas.quantity.Quantitys#create(java.math.BigDecimal, de.metas.product.ProductId)`